Source/OxyPlot/PlotModel/PlotModel.cs

```diff
@@ -128,6 +128,7 @@ public PlotModel()
             this.PlotAreaBorderThickness = new OxyThickness(1);
             this.EdgeRenderingMode = EdgeRenderingMode.Automatic;
 
+            this.AssignColorsToInvisibleSeries = true;
```

Comment:
This will be a breaking change, as in it will potentially change the automatic colors given to series when invisible series are present.
I'm not saying it's necessarily a bad thing, just wanted to consider it. In most cases it is probably preferable to assign colors to invisible series, but one could also imagine situations where this might lead to 'odd' results.

Overall I think it's fine to enable this by default. Maybe the fact that the default behaviour changed could be stressed a bit more in the changelog?

Reply:
If you think it's OK to change the default, then I say let's go for it. Equally, I'd be happy making it not the default for compatibility: my only real interest is in the default for the example browser.

What might you suggest for the changelog? Some explicit indication that this is a silent breaking change?

Reply:
Maybe mention the new `PlotModel.AssignColorsToInvisibleSeries` in the `Added` section and add a separate point in the `Changed` section along the lines of `Enable PlotModel.AssignColorsToInvisibleSeries by default`? I think this would make it more obvious.

But if you have a better idea I'm also happy :)

Reply:
Yeah, might as well add an entry to `Added`. I'll get on with that now.